src/pudl/transform/params/ferc1.py

```diff
@@ -3841,9 +3857,14 @@
         "drop_invalid_rows": [
             {
                 "invalid_values": [pd.NA, np.nan, ""],
-                "required_valid_cols": ["income"],
+                "required_valid_cols": ["dollar_value"],
             },
         ],
+        "check_table_calculations": {
+            "column_to_check": "dollar_value",
+            "calculation_tolerance": 0.08,  # this is still higher than desired.
```

Comment:
This can be dropped now that this table is working as expected.

Reply:
What tolerance should we use now?

Reply:
the default which is used if this isn't set in the params. this line should just be removed.